toolkit/tools/pkg/imagecustomizerlib/partitionutils.go

```diff
@@ -126,6 +126,14 @@ func findRootfsPartition(diskPartitions []diskutils.PartitionInfo, buildDir stri
 			continue
 		}
 
+		// Check if the partition is using DM_verity_hash file system type.
```

Comment:
Please move this check outside of the `findRootfsPartition` function.

Ideally, a function/class/etc should do 1 thing and do it well. You should avoid attaching extra behaviors to things unless there isn't any other good options. In this case, it is pretty trivial to have another `for` loop over the `diskPartitions` list.

Reply:
Update the logic, please check if this logic makes sense.